### PR TITLE
coalib: Add get_version function

### PR DESCRIPTION
**Pull request by sils1297**
To remove duplication.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # Start ignoring PyImportSortBear as imports below may yield syntax errors
-from coalib import assert_supported_version, VERSION, VERSION_FILE, BUS_NAME
+from coalib import assert_supported_version, VERSION, get_version, BUS_NAME
```
Comment (SanketDG) [inline setup.py]: so why import VERSION?

Comment (sils1297) [inline setup.py]: read line 60 and 61

**Comment by SanketDG** [comment]
ack 6b3b163

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**